docs/serving/samples/hello-world/helloworld-php/Dockerfile

```diff
@@ -1,16 +1,10 @@
-# Use the official PHP 7.3 image.
+# Use the official PHP image
 # https://hub.docker.com/_/php
-FROM php:7.3-apache
+FROM php:cli-alpine
 
-# Copy local code to the container image.
-COPY index.php /var/www/html/
-
-# Use the PORT environment variable in Apache configuration files.
-RUN sed -i 's/80/${PORT}/g' /etc/apache2/sites-available/000-default.conf /etc/apache2/ports.conf
-
-# Configure PHP for development.
-# Switch to the production php.ini for production operations.
-# RUN mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini"
-# https://hub.docker.com/_/php#configuration
-RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini"
+COPY index.php /
+EXPOSE 80
 
+# Use PHP built-in web server for serving index.php
+# https://www.php.net/manual/en/features.commandline.webserver.php
+ENTRYPOINT ["php", "-S", "0.0.0.0:80", "/index.php"]
```

Comment:
This should use `$PORT` in the argument to `-S`, rather than hard-coding to 80, which won't work in certain environments.

Reply:
Hmm, I am confused. This is the port for the running docker image / the docker container. Port 80 should always be ok from the images point of view?! 
I have searched for another "hello world" knative serving example applying this $PORT pattern, but did not find one. Can you me an URL?